e2e/e2etests/test_inbound_trackers.go

```diff
@@ -0,0 +1,87 @@
+package e2etests
+
+import (
+	"math/big"
+
+	"github.com/stretchr/testify/require"
+	"github.com/zeta-chain/protocol-contracts/v2/pkg/gatewayevm.sol"
+
+	"github.com/zeta-chain/node/e2e/runner"
+	"github.com/zeta-chain/node/e2e/utils"
+	"github.com/zeta-chain/node/pkg/coin"
+	crosschaintypes "github.com/zeta-chain/node/x/crosschain/types"
+)
+
+// TestInboundTrackers tests inbound trackers processing in ZetaClient
+// It run deposits, send inbound trackers and check cctxs are mined
+// IMPORTANT: the test requires inbound observation to be disabled, the following line should be uncommented:
+// https://github.com/zeta-chain/node/blob/9dcb42729653e033f5ba60a77dc37e5e19b092ad/zetaclient/chains/evm/observer/inbound.go#L210
+func TestInboundTrackers(r *runner.E2ERunner, args []string) {
+	require.Len(r, args, 0)
+
+	amount := big.NewInt(1e17)
+
+	addTrackerAndWaitForCCTX := func(coinType coin.CoinType, txHash string) {
+		r.AddInboundTracker(coinType, txHash)
+		cctx := utils.WaitCctxMinedByInboundHash(r.Ctx, txHash, r.CctxClient, r.Logger, r.CctxTimeout)
+		require.EqualValues(r, crosschaintypes.CctxStatus_OutboundMined, cctx.CctxStatus.Status)
+		r.Logger.CCTX(*cctx, "cctx")
+	}
+
+	// send v1 eth deposit
+	r.Logger.Print("🏃test v1 eth deposit")
```

Comment:
It's only run in the CLI for now, I think it looks nicer to print the different test run. Otherwise you would need to use verbose that also print all other details